server/ldap_check.rb

```diff
@@ -43,27 +48,45 @@ def initialize(args={})
       @conf_values[k] = args[k]
     }
 
+    configuration['cache_seconds'] = @default_cache_seconds unless (configuration['cache_seconds'])
+
     if logger.debug?
       logger.debug "final conf values"
-      @conf_values.each_key { |k| logger.debug "key: [#{k}] value: [#{@conf_values[k]}]" }
+      configuration.each_key { |k| logger.debug "key: [#{k}] value: [#{configuration[k]}]" }
```

Comment:
This was really confusing to me -- I had to look at the source code to realize that configuration is an access method for @conf_values. I think a consistent access method would be less likely to cause future problems and easier to understand?